src/components/admin/Api/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Divider, Grid, SxProps, Theme, Typography } from '@mui/material';
+import { Auth } from '@supabase/ui';
+import { authenticatedRoutes } from 'app/routes';
+import AdminTabs from 'components/admin/Tabs';
+import SingleLineCode from 'components/content/SingleLineCode';
+import usePageTitle from 'hooks/usePageTitle';
+import { FormattedMessage } from 'react-intl';
+import RefreshToken from './RefreshToken';
+
+const boxStyling: SxProps<Theme> = {
+    padding: 2,
+};
+
+function AdminApi() {
+    usePageTitle({
+        header: authenticatedRoutes.admin.api.title,
+        headerLink:
+            'https://docs.estuary.dev/reference/authentication/#authenticating-flow-using-the-cli',
```

Comment:
This should be removed and this link should move down to be next to the `Access Token` header like `Cloud Storage` does it.